deployment/ConnectedVehicleARMTemplate.json

```diff
@@ -75,6 +75,12 @@
                 "description": "Claims provider uri which is used for platformAccount deployment."
             }
         },
+        "claimsProviderApiVersion": {
+            "type": "string",
```

Comment:
I'd suggest adding an allowed values property to the parameters so it's clear to the customer what the currently supported api versions are. Same in the other ARM template
```
      "allowedValues": [
        "2021-05-19",
        "2022-01-30-preview"
      ],
```